src/h5web/visualizations/matrix/Cell.tsx

```diff
@@ -13,13 +13,14 @@ function Cell(props: GridChildComponentProps): JSX.Element {
 
   // -1 to account for the index row and column
   const dataValue = valueAccessor(rowIndex - 1, columnIndex - 1);
+
   return (
     <div
       className={styles.cell}
       style={style}
       data-bg={(rowIndex + columnIndex) % 2 === 1 ? '' : undefined}
     >
-      {format('.3e')(dataValue)}
+      {typeof dataValue === 'number' ? format('.3e')(dataValue) : dataValue}
```

Comment:
What prompted this ?

Reply:
`MatrixVis` has a prop of type `ndarray<number | string>`. Why? Because of its support predicate:

```ts
isBaseType(type) && isSimpleShape(shape) && shape.dims.length >= 1
```

`isBaseType` checks for `[HDF5TypeClass.Integer, HDF5TypeClass.Float, HDF5TypeClass.String]`

Reply:
So we now support visualizing arrays of strings with `MatrixVis`.

Reply:
All right :+1: